### PR TITLE
[Workflow] Add tip for workflow configuration default values

### PR DESCRIPTION
**Pull request by dominikhajduk**
Added tip for http://symfony.com/doc/current/workflow/usage.html configuration.

Tested and based on great examples provided by @lyrixx in: https://github.com/lyrixx/SFLive-Paris2016-Workflow/blob/master/app/config/workflow.yml#L3 and
https://github.com/lyrixx/SFLive-Paris2016-Workflow/blob/master/src/AppBundle/Entity/Article.php#L24

### CODE REVIEW

**Review by javiereguiluz — CHANGES_REQUESTED** [review]
Adding this tip looks like a good idea to me. Sadly, it's hard to understand it, so I'd recommend to reword it a bit. Thanks!

**Comment by dominikhajduk** [comment]
@javiereguiluz thanks for CR. I've rewritten tip to be clearer in the same PR. If you have additional suggestions please let me know. Or should I open new PR?

**Comment by javiereguiluz** [comment]
@dominikhajduk thanks for this reword. I can now understand everything perfectly 👍

**Review by javiereguiluz — APPROVED** [review]
👍

File: workflow/usage.rst [hunk 1]
```diff
@@ -148,6 +148,12 @@ like this:
     The marking store type could be "multiple_state" or "single_state".
     A single state marking store does not support a model being on multiple places
     at the same time.
+    
+.. tip::
+
+    Workflow configuration ``type`` and ``arguments`` in ``marking_store`` section 
+    are optional. You can skip one or both when you use default values. Default value
+    for ``type`` is "single_state". Default value for ``arguments`` is "marking".
```
Comment (xabbuh) [inline workflow/usage.rst]: I would reword this a bit to something like this:

> The `type` (default value `single_state`) and `arguments` (default value `marking`) attributes of the `marking_store` option are optional. If omitted, their default values will be used.

What do you think?

Comment (dominikhajduk) [inline workflow/usage.rst]: @xabbuh fine for me. Thanks.

**Comment by dominikhajduk** [comment]
@xabbuh should be fine now. Thanks

**Comment by xabbuh** [comment]
:+1:

@javiereguiluz Good for you too?

**Comment by javiereguiluz** [comment]
@xabbuh yes, it looks very nice!

**Comment by xabbuh** [comment]
Thank you Dominik.
